tests/dualtor/test_tunnel_memory_leak.py

```diff
@@ -21,15 +21,22 @@
 from tests.common.dualtor.dual_tor_utils import build_packet_to_server
 from tests.common.dualtor.dual_tor_utils import flush_neighbor
 from tests.common.helpers.dut_utils import get_program_info
+from tests.common.fixtures.ptfhost_utils import run_garp_service, run_icmp_responder
 
 
 pytestmark = [
-    pytest.mark.topology("dualtor")
+    pytest.mark.topology("dualtor"),
+    pytest.mark.usefixtures('run_garp_service',
+                            'run_arp_responder',
+                            'run_icmp_responder'
```

Comment:
It's unnecessary to run `usefixtures` for `run_garp_service` and `run_icmp_responder` because these two fixtures are autoused.

Reply:
Thanks to point it out.  Fixed it.